frontend/src/components/admin-page/actions-button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import downloadIcon from 'assets/images/admin-page/vectordownload.svg';
+import pdfIcon from 'assets/images/admin-page/vectorpdf.svg';
+import editIcon from 'assets/images/admin-page/vectoredit.svg';
+import historyIcon from 'assets/images/admin-page/vectorhistory.svg';
+import styles from './styles.module.scss';
+
+interface IProps {
+  edit: (id: string) => void,
+  deleteUser: (id:string)=> void,
+  id: string,
+}
+
+function ActionsButton({ edit, id, deleteUser }: IProps) {
+  return (
+    <div className={styles.iconsDiv}>
+      <div onClick={() => edit(id)} className={styles.iconDiv}>
+        <img src={editIcon} />
```

Comment:
Always add alt, width, height, loading  attrs to the `img`
By the way, it would be better to use this icons as background image